test/unittest/statistics/dds/CMakeLists.txt

```diff
@@ -22,7 +22,9 @@ if(NOT ((MSVC OR MSVC_IDE) AND EPROSIMA_INSTALLER))
         endif()
 
         set(STATISTICS_DOMAINPARTICIPANT_TESTS_SOURCE
-            StatisticsDomainParticipantTests.cpp)
+            StatisticsDomainParticipantTests.cpp
+			${PROJECT_SOURCE_DIR}/src/cpp/statistics/types/typesPubSubTypes.cxx
+			${PROJECT_SOURCE_DIR}/src/cpp/statistics/types/types.cxx)
```

Comment:
```suggestion
            StatisticsDomainParticipantTests.cpp
            ${PROJECT_SOURCE_DIR}/src/cpp/statistics/types/typesPubSubTypes.cxx
            ${PROJECT_SOURCE_DIR}/src/cpp/statistics/types/types.cxx)
```

Reply:
Done in 19de4fefdde91a41f08bbeced28e9d2d8b475c8b